### ISSUE TITLE
The new supplementary synthetic dataset

### ISSUE THREAD

**Issue by velueers** [feihuzhang/DSMNet#4]
Thank you very much again for your sharing. May I venture to ask if the new supplementary synthetic dataset can be Shared?

**Comment by feihuzhang** [comment]
Yes. Will be available in this month. I am finding the online storage for it.

**Comment by french-paragon** [comment]
Do you have a link for the new synthetic dataset ?
